ios/PurchasesHybridCommon/PurchasesHybridCommon/StoreProduct+HybridAdditions.swift

```diff
@@ -21,27 +21,40 @@ internal extension StoreProduct {
             "introPrice": NSNull(),
             "price": self.price,
             "priceString": self.localizedPriceString,
+            "pricePerWeek": NSNull(),
+            "pricePerMonth": NSNull(),
+            "pricePerYear": NSNull(),
+            "pricePerWeekString": NSNull(),
+            "pricePerMonthString": NSNull(),
+            "pricePerYearString": NSNull(),
             "productCategory": self.productCategoryString,
             "productType": self.productTypeString,
             "title": self.localizedTitle,
             "subscriptionPeriod": NSNull(),
         ]
 
-        if #available(iOS 11.2, tvOS 11.2, macOS 10.13.2, *),
-           let introductoryDiscount = self.introductoryDiscount {
-            dictionary["introPrice"] = introductoryDiscount.rc_dictionary
-        }
+        if #available(iOS 11.2, tvOS 11.2, macOS 10.13.2, *) {
+            dictionary["pricePerWeek"] = self.pricePerWeek
+            dictionary["pricePerMonth"] = self.pricePerMonth
+            dictionary["pricePerYear"] = self.pricePerYear
+            dictionary["pricePerWeekString"] = self.localizedPricePerWeek
+            dictionary["pricePerMonthString"] = self.localizedPricePerMonth
+            dictionary["pricePerYearString"] = self.localizedPricePerYear
 
-        if #available(iOS 12.2, tvOS 12.2, macOS 10.14.4, *) {
-            dictionary["discounts"] = self.discounts.map { $0.rc_dictionary }
-        }
+            if let introductoryDiscount = self.introductoryDiscount {
+                dictionary["introPrice"] = introductoryDiscount.rc_dictionary
+            }
 
-        if #available(iOS 11.2, macOS 10.13.2, tvOS 11.2, watchOS 6.2, *) {
-            if let subscriptionPeriod = self.subscriptionPeriod {
-                dictionary["subscriptionPeriod"] = StoreProduct.rc_normalized(subscriptionPeriod: subscriptionPeriod)
+            if #available(iOS 12.2, tvOS 12.2, macOS 10.14.4, *) {
+                dictionary["discounts"] = self.discounts.map { $0.rc_dictionary }
             }
-        }
 
+            if #available(watchOS 6.2, *) {
```

Comment:
Lint complained about the check that was previously here: 
```swift
if #available(iOS 11.2, macOS 10.13.2, tvOS 11.2, watchOS 6.2, *)
```
saying that `iOS` was already covered by the enclosing check a few lines up:
```swift
if #available(iOS 11.2, tvOS 11.2, macOS 10.13.2, *)
```
I figured that Lint warning is also valid for `macOS` and `tvOS`, so I kept only `watchOS`.

I'm not 100% sure this is correct, so please double check.

Reply:
Hmm I don't know why there is a requirement for watchOS on this one TBH... But this seems the best way to avoid a breaking change. Maybe we could try to remove the checks and try on an older emulator...